api/v1beta1/foundationdbcluster_types.go

```diff
@@ -1184,13 +1185,30 @@ type FoundationDBClusterAutomationOptions struct {
 	MaxConcurrentReplacements *int `json:"maxConcurrentReplacements,omitempty"`
 
 	// DeletionMode defines the deletion mode for this cluster. This can be
-	// DeletionModeAll, DeletionModeZone or DeletionModeProcessGroup. The
+	// PodUpdateModeAll, PodUpdateModeZone or PodUpdateModeProcessGroup. The
 	// DeletionMode defines how Pods are deleted in order to update them or
 	// when they are removed.
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:validation:Enum=All;Zone;ProcessGroup
+	// +kubebuilder:validation:Enum=All;Zone;ProcessGroup;None
 	// +kubebuilder:default:=Zone
-	DeletionMode DeletionMode `json:"deletionMode,omitempty"`
+	DeletionMode PodUpdateMode `json:"deletionMode,omitempty"`
+
+	// RemovalMode defines the removal mode for this cluster. This can be
+	// PodUpdateModeAll, PodUpdateModeZone or PodUpdateModeProcessGroup. The
+	// RemovalMode defines how process groups are deleted in order when they
+	// are marked for removal.
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Enum=All;Zone;ProcessGroup;None
+	// +kubebuilder:default:=Zone
+	RemovalMode PodUpdateMode `json:"removalMode,omitempty"`
+
+	// WaitTimeBetweenRemovals defines how long to wait between the last removal and the next removal. This is only an
+	// upper limit if the process group and the according resources are deleted faster then the provided duration the
+	// operator will move on with the next removal. The idea is to prevent a race condition were the operator deletes
+	// a resource but the Kubernetes API is slower to trigger the actual deletion and we are running into a situation
+	// where the fault tolerance check still includes the already deleted processes.
+	// Defaults to 60s.
+	WaitTimeBetweenRemovals time.Duration `json:"waitTimeBetweenRemovals,omitempty"`
```

Comment:
Should we include the unit in the field name? The fact that it's parsed into a duration tells us how it's interpreted on the go side, but it could be ambiguous from the perspective of clients.

Reply:
That's a good idea 👍